src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -14,6 +14,9 @@
 
 package com.google.codeu.data;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
```

Comment:
Please remove unrelated changes like this and the modifications to the Query lines, comments, and the license text below. It appears that you are using some code formatting feature that is making unnecessary changes. Please use "mvn com.coveo:fmt-maven-plugin:format" to format the code instead of what you are using right now. 

I believe if you run "mvn com.coveo:fmt-maven-plugin:format" it should fix everything but the changes to the license text for you.

Reply:
I think now it should be good

Reply:
I think you are now importing this twice, lines 17-19 and lines 30-32
If so, can you keep the original version, in lines 30-32?

Reply:
I think you are now importing this twice, lines 17-19 and lines 30-32
If so, can you keep the original version, in lines 30-32?

Reply:
You're right, done